src/rules/number-zero-length-no-unit/README.md

```diff
@@ -0,0 +1,35 @@
+# number-zero-length-no-unit
+
+Disallow units for lengths of zero.
+
+```css
+    a { top: 0px; }
+/**          ↑ ↑
+ * This zero and this type of length unit */
+```
+
+For zero lengths the unit identifier is optional.
```

Comment:
We should probably have a brief explanation of how a "length" differs from other value types, why `0%` and `0s` are accepted. We might even list out the units that will be checked?